### ISSUE TITLE
Add a way to pass arguments to the pager via a command line flag

### ISSUE THREAD

**Issue by cohml** [sharkdp/bat#2837]
Let's say you set a custom pager like this:

```bash
BAT_PAGER="less --tabs 4 -RF"
```

Then behind the scenes, those arguments will be passed into the specified pager program when paginating `bat`. This therefore "hardcodes" the "default" behavior of your pager when using `bat`.

But sometimes it's useful to one-off modify this default pager behavior. For example, to preload a search query like `less +/some\ string`.

Currently, I think the only way to do this would be as follows:

```bash
BAT_PAGER="${BAT_PAGER} +/some\ string" bat ...
```

But that is quite painful and verbose, not to mention potentially brittle since it depends on the user remembering the exact value of their `BAT_PAGER`.

So what I propose therefore is a way to pass extra args via a command line argument, e.g.

```bash
bat --pager-opts="+/some\ string" ...
```

Related issues:
- #352
- #2426

**Comment by eth-p** [comment]
This is already possible through the `--pager` option :)

```console
$ bat src/printer.rs --pager='less --version'
less 643 (PCRE2 regular expressions)
Copyright (C) 1984-2023  Mark Nudelman

less comes with NO WARRANTY, to the extent permitted by law.
For information about the terms of redistribution,
see the file named README in the less distribution.
Home page: https://greenwoodsoftware.com/less
```

You can even use arguments with spaces by quoting them:

```bash
bat src/printer.rs --pager='less "+/&mut OutputHandle"'
```

I'm not too keen on the idea of adding a separate `--pager-opts` option for this, though. It's the same number of characters to type  `--pager-opts='...'` as it is to type `--pager='less ...'`, so we would be making `bat` more complex without saving any keystrokes for the user in doing so.

**Comment by cohml** [comment]
But to be clear, this

```bash
bat src/printer.rs --pager='less "+/&mut OutputHandle"'
```

is not the same as this

```bash
BAT_PAGER="${BAT_PAGER} +/&mut OutputHandle" bat src/printer.rs
```

The former *overwrites* whatever you've set for `BAT_PAGER`. The latter - which is the use case for which I propose `--pager-opts` (we could probably find a clearer name) - would *append* to it.

If most people set `BAT_PAGER` to something super simple like `less -RF`, then sure, the value add of `--pager-opts` isn't huge. But I personally pimp out my `BAT_PAGER`, and surely I'm not the only one:

```bash
❯ echo $BAT_PAGER
less --chop-long-lines --RAW-CONTROL-CHARS --header=1 --jump-target=3 --line-num-width=0 --LONG-PROMPT --no-number-headers --shift=4 --status-column --use-color --color=HGk --color=NGk
```

Using `--pager` as you propose completely wipes out all of that. So adding a way to basically add extra args onto the current value would be a nice (if somewhat niche, I admit) addition.

**Comment by cohml** [comment]
> That is quite the customization, wow.

Haha, thanks. I'm quite proud of it :) `less` is amazing. (As is `bat`!)

To your question, those are `less`-specific options. But to me, `less` IS `bat` and vice-versa. I only ever invoke `bat`, and page basically everything. `bat` is essentially my pager.

My case is perhaps a bit extreme, I don't know. But I still think that more generally a nice case could be made for some way to pass arguments directly to the pager program via the `bat` API. Especially for people like me who use `bat` for basically everything and a powerful pager like `less`, such an option would de facto grandfather in all kinds of nice functionality into the `bat` experience.

**TL;DR:** They're `less` options, but I would wholeheartedly support that PR at any point in time because I think it would be of wider utility. I'd open one it myself if I only knew Rust 😭 

Thanks for the quick replies by the way!
